examples/send-multiple-l2-tx/main.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"log"
+	"math/big"
+
+	"github.com/hermeznetwork/hermez-go-sdk/account"
+	"github.com/hermeznetwork/hermez-go-sdk/client"
+	"github.com/hermeznetwork/hermez-go-sdk/node"
+	"github.com/hermeznetwork/hermez-go-sdk/transaction"
+	hezCommon "github.com/hermeznetwork/hermez-node/common"
+)
+
+const (
+	ethereumNodeURL = ""
+	chainID         = 5 // set this accordingly to the ethereum node
+
+	sourceAccPvtKey           = ""
+	auctionContractAddressHex = "0x1D5c3Dd2003118743D596D7DB7EA07de6C90fB20"
+)
+
+var (
+	hezToken = hezCommon.Token{
+		TokenID: 1,
+		Symbol:  "HEZ",
+	}
+	txsMetadata = []txMetadata{
+		{"0xb48cA794d49EeC406A5dD2c547717e37b5952a83", 126, big.NewInt(1100000000000000000)},
+		{"0x263C3Ab7E4832eDF623fBdD66ACee71c028Ff591", 126, big.NewInt(1200000000000000000)},
+		{"0xb8eD2B0a6e17649c9cE891895D3D9297Ab448f03", 126, big.NewInt(1300000000000000000)},
+		{"0x4E857Ac4A07cAD0B50CD006158f5E5A521A880CE", 126, big.NewInt(1400000000000000000)},
+	}
+)
+
+type txMetadata struct {
```

Comment:
This is nice. Perhaps it would be good to be public and to stay at transacations/model.go